contracts/apps/WitnetRandomnessV20.sol

```diff
@@ -0,0 +1,500 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0 <0.9.0;
+
+import "../WitnetRandomness.sol";
+import "../apps/UsingWitnet.sol";
+import "../interfaces/IWitnetRandomnessAdmin.sol";
+import "../patterns/Ownable2Step.sol";
+
+/// @title WitnetRandomnessV20: Unmalleable and provably-fair randomness generation based on the Witnet Oracle.
+/// @author The Witnet Foundation.
+contract WitnetRandomnessV20
```

Comment:
I'd suggest using v2 instead of v20 to make it easier to read
```suggestion
contract WitnetRandomnessV2
```

Reply:
Data model is determined by first two semver digits. With V20 we're clearly stating that this bypass contract binds branch's version (0.7.x) to v2.0. Former legacy addresses have already been refered as V06 or V07.